entity-framework/core/extensions/index.md

```diff
@@ -72,6 +72,12 @@ With Entity Framework Core query plan debugger visualizer, you can view the quer
 
 ## Extensions
 
+### AutoCompute
```

Comment:
Suggest adding it to the bottom of the list, not the top

Reply:
Sounds good, I will adjust it. I thought it was alphabetically sorted.

Reply:
Fair assumption, but it is completely randon at the moment.

Reply:
Updated